### DISCUSSION TITLE
Questions Regarding Log Storage and Node Management in Security Onion

### DISCUSSION THREAD

**Discussion by SankaGamage** [ory/kratos#4258] [Q&A] [answered]
Hello everyone,

I recently installed Security Onion on a server and accessed it via VPN through the web interface. I have a few questions regarding log storage and node management that I would like to clarify:

1. How long are logs typically stored in Security Onion by default?
2. Is it possible to change the log retention period? If so, how can this be configured?
3. Where are logs normally stored in Security Onion?
4. If logs are overwritten, how many days of logs are retained before they are overwritten?
5. Is it possible to store logs from different branches or data sources separately?
6. If I want to add additional nodes to my Security Onion setup, what are the steps to do so, and what challenges might I face during the process?

Any help or insights you can provide would be greatly appreciated. Thank you!

**Comment by vinckr** [comment, marked as answer]
Hello @PGDasunMadhusanka 

Please head over to the [security onion repository](https://github.com/Security-Onion-Solutions/securityonion/discussions) or to [their website](https://securityonionsolutions.com/) - this discussion forum is for Ory related questions strictly.
Thanks
